raiden/tests/unit/test_wal.py

```diff
@@ -125,3 +134,26 @@ def test_write_read_events():
     latest_event = new_events[-1]
     assert latest_event[0] == block_number
     assert isinstance(latest_event[1], EventTransferSentFailed)
+
+
+def test_restore_without_snapshot():
+    wal = new_wal()
+
+    wal.log_and_dispatch(Block(5), 5)
+    wal.log_and_dispatch(Block(7), 7)
+    wal.log_and_dispatch(Block(8), 8)
+
+    newwal, events = restore_from_latest_snapshot(
+        state_transtion_acc,
+        wal.storage,
+    )
+
+    assert not events
+
+    aggregate = newwal.state_manager.current_state
+    assert aggregate.state_changes == [Block(5), Block(7), Block(8)]
+
+
+class AccState(State):
```

Comment:
I'd move that above `state_transition_acc`.

Reply:
I usually follow the order:

- imports
- constants
- log
- functions
- classes

on the code I write, that's why I added this class in the end of the file. but I will move it up if that helps readability.

Reply:
Was just a suggestion, I like to keep such stuff together.